src/aiortc/contrib/media.py

```diff
@@ -230,14 +280,15 @@ class MediaPlayer:
     :param loop: Whether to repeat playback indefinitely (requires a seekable file).
     """
 
-    def __init__(self, file, format=None, options={}, loop=False):
+    def __init__(self, file, format=None, options={}, loop=False, transcode=True):
```

Comment:
I'm not sure "transcode" is descriptive. The media player is not transcoding anything, it's decoding frames.

Reply:
I used transcode because the  original behavior is:
1 - get stream and **decode** in frames
2 - Get frames and **encode** to packet in specific codec (h264, vp8... depends

In my modification, it does not `decode->encode` frame (`transcode`). So you have a suggestion for this argument? Please, tell me a better name for this.

Reply:
Again, the MediaPlayer is not *encoding* anything, so your flag is purely controlling whether you want to perform decoding or not, i.e. if you want Packet or Frame instances in the output.

Reply:
I'll change to `decode`. Is it ok?